alchemist-loading/src/test/resources/yamlAliases/aliases.yml

```diff
@@ -0,0 +1,79 @@
+incarnation: protelis
+
+network-model:
+  type: ConnectWithinDistance
+  parameters: [5]
+
+_node: &node
+  - type: Point
+    parameters: [0, 0]
+
+deployments:
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
+  - *node
```

Comment:
```suggestion
_node: &node
  - type: Point
    parameters: [0, 0]

_nodes: &nodes
  [*node, *node, *node, *node, *node, *node, *node, *node, *node, *node]

_moar_nodes: &moar
  [*nodes, *nodes, *nodes, *nodes, *nodes, *nodes, *nodes, *nodes, *nodes, *nodes]

_recur: &recur
  [*moar, *moar, *moar, *moar, *moar, *moar, *moar, *moar, *moar, *moar]

deployments:
  [*recur, *recur, *recur, *recur, *recur, *recur, *recur, *recur, *recur, *recur]
```

Reply:
This way the number of nodes deployed is esponential

Reply:
![image](https://user-images.githubusercontent.com/1991673/177180551-737d8569-e3d1-4998-bc3c-c2a3cd0e125c.png)

which makes it a good stress test, doesn't it?

Reply:
In order to test the previous limit for aliases (50), the deployment should contain more than 100,000 nodes.
Is that okay?

Reply:
I don't think I fully understand.
The feature I want to test with this simulation concerns the number of times I can refer to an anchor in YAML. In previous versions of the simulator, the limit was 50. I'm now testing a simulation with more than 50 aliases. The number of nodes deployed is not really correlated to that.

In the last commit I used 51 aliases, deploying ~4000 nodes.

Reply:
I misunderstood the reason the limit is in. To me, this:

```yaml
a: &a ["lol","lol","lol","lol","lol","lol","lol","lol","lol"]
b: &b [*a,*a,*a,*a,*a,*a,*a,*a,*a]
c: [*b,*b,*b,*b,*b,*b,*b,*b,*b]
```
should cause 10(1 + 10) expansions, although with "just" 20 alias calls. I expected the count to be performed recursively, but it is actually not, as the original problem is avoiding recursive calls. Please discard my previous comment.